apps/files/tasks.py

```diff
@@ -31,8 +38,133 @@ def extract_file(viewer, **kw):
                      (viewer, err))
         else:
             msg.save(_('There was an error accessing file %s.') % viewer)
-        task_log.error('[1@%s] Error unzipping: %s' %
-                       (extract_file.rate_limit, err))
+        log.error('[1@%s] Error unzipping: %s' %
```

Comment:
Not a fan of this format, but we're discussing that in https://github.com/mozilla/olympia/issues/975